corehq/apps/app_manager/views/form_designer.py

```diff
@@ -76,6 +76,19 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
     })
 
+    scheduler_data_nodes = []
+    if getattr(module, 'has_schedule', False):
+        scheduler_data_nodes = [
+            "unscheduled_visit",
+            "current_visit_number",
+            "next_visit",
+            "next_visit_date",
+            "next_due",
+        ]
```

Comment:
That `next_visit` was an error in confluence. Removed it, so the list should be good now. Sorry for the confusion. Your list looks good.